cmd/kubeadm/app/cmd/config.go

```diff
@@ -91,6 +92,61 @@ func NewCmdConfig(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+// NewCmdConfigPrint returns cobra.Command for "kubeadm config print" command
+func NewCmdConfigPrint(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "print",
+		Short: "Print configuration",
+		Long:  "This command serves as a wrapper for subcommands that are meant to print configuration or part of it.",
```

Comment:
optionally:
`meant to print configuration or part of it.`
->
`meant to print the default configuration or part of it.`

Reply:
"This command prints default configurations for subcommands provided"

Reply:
Updated with Tim's version, but omitted the "default" word from it as `kubectl config print` may also be used to print non default configs in the future (such as current ClusterConfiguration from config map, etc.)